frontend-web/webclient/app/ui-components/Button.tsx

```diff
@@ -102,6 +103,11 @@ export const ButtonClass = injectStyle("button", k => `
         border-bottom-left-radius: 0;
     }
 
+    ${k}[data-attached-left=true] {
+        border-top-right-radius: 0;
+        border-bottom-right-radius: 0;
+    }
```

Comment:
Does this mean this is attached to the left of the button?

Also, if `attached-left` exists, attached should be renamed to `attached-right`